scripts/src/java/org/oppia/android/scripts/maven/GenerateMavenDependenciesList.kt

```diff
@@ -91,15 +90,63 @@ class GenerateMavenDependenciesList(
 
     val licensesToBeFixed = getAllBrokenLicenses(finalDependenciesList)
 
-    // TODO(#3486): Update GenerateMavenDependenciesList.kt to call out first coordinate name
-    // that should be updated to update all occurrences of the license.
     if (licensesToBeFixed.isNotEmpty()) {
-      println("\nPlease verify the license link(s) for the following license(s) manually:")
+      val licenseToDependencyMap = findFirstDependencyWithBrokenLicense(
+        finalDependenciesList,
+        licensesToBeFixed
+      )
+      println(
+        """
+        Some licenses do not have their 'original_link' verified. To verify a license link, click
+        on the original link of the license and check if the link points to any valid license or
+        not. If the link does not point to a valid license (e.g - https://fabric.io/terms), set 
+        the 'is_original_link_invalid' field of the license to 'true'.
+         
+        e.g - 
+        license {
+          license_name: "Terms of Service for Firebase Services"
+          original_link: "https://fabric.io/terms"
+          is_original_link_invalid: true
+        }
+  
+        If the link does point to a valid license then choose the most appropriate category for 
+        the link:
+        1. scrapable_link: If the license text is plain text and the URL mentioned can be scraped
+                           directly from the original_link of the license. 
+                           e.g - https://www.apache.org/licenses/LICENSE-2.0.txt
+        2. extracted_copy_link: If the license text is plain text but it can not be scraped 
+                                directly from the original_link of the license.
+                                e.g - https://www.opensource.org/licenses/bsd-license
+        3. direct_link_only: If the license text is not plain text, it's best to display only the
+                             link of the license.
+                             e.g - https://developer.android.com/studio/terms.html
+        
+        After identifying the category of the license, modify the license to include one of the
+        above mentioned 'url'. 
+        
+        e.g - 
+        license {
+          license_name: "The Apache Software License, Version 2.0"
+          original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+          scrapable_link {
+            url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+          }
+        }
+        
+        Please verify the license link(s) for the following license(s) manually in 
+        maven_dependencies.textproto, note that only first dependency that contains the license 
+        needs to be updated and also re-run the script to update the license details at all places:
```

Comment:
Why is `:` at the end?

Reply:
Looks odd when we read the entire sentence and therefore expecting a `.`

Reply:
Done.